ui/page/claim/internal/claimPageComponent/internal/channelPage/tabs/contentTab/internal/searchResults.jsx

```diff
@@ -2,24 +2,37 @@
 import React from 'react';
 import ClaimList from 'component/claimList';
 import { DEBOUNCE_WAIT_DURATION_MS, SEARCH_PAGE_SIZE } from 'constants/search';
+import * as CS from 'constants/claim_search';
 import { lighthouse } from 'redux/actions/search';
 
 type Props = {
   searchQuery: string,
   claimId: ?string,
   showMature: ?boolean,
   tileLayout: boolean,
+  orderBy?: ?string,
   onResults?: (results: ?Array<string>) => void,
   doResolveUris: (Array<string>, boolean) => void,
 };
 
 export function SearchResults(props: Props) {
-  const { searchQuery, claimId, showMature, tileLayout, onResults, doResolveUris } = props;
+  const { searchQuery, claimId, showMature, tileLayout, orderBy, onResults, doResolveUris } = props;
 
   const [page, setPage] = React.useState(1);
   const [searchResults, setSearchResults] = React.useState(undefined);
   const [isSearching, setIsSearching] = React.useState(false);
   const noMoreResults = React.useRef(false);
+  const [sortBy, setSortBy] = React.useState(1);
+
+  React.useEffect(() => {
+    setSortBy(
+      !orderBy || orderBy === CS.ORDER_BY_TRENDING
+        ? ``
+        : orderBy === CS.ORDER_BY_TOP
+        ? `&sort_by=${CS.ORDER_BY_TOP_VALUE[0]}`
+        : `&sort_by=${CS.ORDER_BY_NEW_VALUE[0]}`
+    );
+  }, [orderBy]);
```

Comment:
`useState` is unnecessary/incorrect here since this is just prop-derivation, and the effect just makes it delayed by 1 cycle.

Use `const sortBy = orderBy || orde ...`